tee-worker/core-primitives/sgx-runtime-primitives/src/types.rs

```diff
@@ -21,7 +21,7 @@ use sp_runtime::{
 	MultiSignature, OpaqueExtrinsic,
 };
 
-use parentchain_primitives::AccountId as ParentchainAccountId;
+use litentry_primitives::AccountId as ParentchainAccountId;
```

Comment:
I think it's better to set this alias inside `litentry_primitives`, like:
```
pub use parentchain_primitives::AccountId as ParentchainAccountId
```
similar to BlockNumber